specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -0,0 +1,2448 @@
+      ],
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "Response by document",
+          "items": {
+            "$ref": "#/definitions/DocumentHealthcareEntities"
+          }
+        },
+        "errors": {
+          "type": "array",
+          "description": "Errors by document id.",
+          "items": {
+            "$ref": "#/definitions/DocumentError"
+          }
+        },
+        "statistics": {
+          "$ref": "#/definitions/RequestStatistics"
+        },
+        "modelVersion": {
+          "type": "string",
+          "description": "This field indicates which model is used for scoring."
+        }
+      }
+    },
+    "DocumentHealthcareEntities": {
+      "type": "object",
+      "required": [
+        "id",
+        "entities",
+        "relations",
+        "warnings"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique, non-empty document identifier."
+        },
+        "entities": {
+          "description": "Healthcare entities.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthcareEntity"
+          }
+        },
+        "relations": {
+          "type": "array",
+          "description": "Healthcare entity relations.",
+          "items": {
+            "$ref": "#/definitions/HealthcareRelation"
+          }
+        },
+        "warnings": {
+          "type": "array",
+          "description": "Warnings encountered while processing document.",
+          "items": {
+            "$ref": "#/definitions/TextAnalyticsWarning"
+          }
+        },
+        "statistics": {
+          "description": "if showStats=true was specified in the request this field will contain information about the document payload.",
+          "$ref": "#/definitions/DocumentStatistics"
+        }
+      }
+    },
+    "HealthcareEntity": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Entity"
+        },
+        {
+          "properties": {
+            "assertion": {
+              "type": "object",
+              "$ref": "#/definitions/HealthcareAssertion"
+            },
+            "name": {
+              "description": "Preferred name for the entity. Example: 'histologically' would have a 'name' of 'histologic'.",
+              "type": "string"
+            },
+            "links": {
+              "description": "Entity references in known data sources.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/HealthcareEntityLink"
+              }
+            }
+          }
+        }
+      ]
+    },
+    "HealthcareRelation": {
+      "type": "object",
+      "description": "Every relation is an entity graph of a certain entityType, where all entities are connected and have specific roles within the relation context.",
+      "required": [
+        "relationType",
```

Comment:
Might be too late for this preview but I believe for the next API version we might need to change this to `relationCategory`. In the past we refrained from using `type` because of its program language context.

Reply:
Agreed. Also - since it is a property of the relation object, does it really need to be relationCategory instead of simply 'category' (similar to the entity object in the response)?